tape/src/main/java/com/squareup/tape2/FileObjectQueue.java

```diff
@@ -32,15 +32,15 @@
   }
 
   @Override public void add(T entry) throws IOException {
-    bytes.reset();
-    converter.toStream(entry, bytes);
-    queueFile.add(bytes.getArray(), 0, bytes.size());
+    converter.toStream(entry, buffer);
+    long size = buffer.size();
+    queueFile.add(buffer.readByteArray(), 0, (int) size);
```

Comment:
You can just call `readByteArray()` and then use the array length for the size

Reply:
yeah. didn't know which style looked better. will switch!